verification/src/block_verifier.rs

```diff
@@ -75,9 +69,6 @@ impl<CP: ChainProvider + Clone> CellbaseVerifier<CP> {
     }
 
     pub fn verify(&self, block: &Block) -> Result<(), Error> {
-        if block.commit_transactions().is_empty() {
```

Comment:
Is cellbase required now? Has it been added in the verification RFC doc?

Reply:
Cellbase is always required. RFC updated .